railties/lib/rails/generators/rails/db/system/change/change_generator.rb

```diff
@@ -41,6 +41,8 @@ def edit_gemfile
           end
 
           def edit_dockerfile
+            return unless File.exist?("Dockerfile")
```

Comment:
I think this needs to be expanded to an absolute path or inside an `in_root` block.

Reply:
is this not consistent with the path that `gsub_file` takes? Both are relative paths.

Reply:
Thanks for the quick review! I updated it to use `File.expand_path("Dockerfile", destination_root)` which is what Thor uses in `gsub_file`: 

https://github.com/rails/thor/blob/044230754cd7ad767f886f8fcb876742605188e0/lib/thor/actions/file_manipulation.rb#L268

Reply:
:+1: My mistake -- I was referencing an old railties implementation of `gsub_file`